modules/logging-release-notes-5-8-0.adoc

```diff
@@ -0,0 +1,56 @@
+//module included in logging-5-8-release-notes.adoc
+:content-type: REFERENCE
+[id="logging-release-notes-5-8-0_{context}"]
+= Logging 5.8.0
+This release includes link:https://access.redhat.com/errata/RHBA-2023:6139[OpenShift Logging Bug Fix Release 5.8.0] and link:https://access.redhat.com/errata/RHBA-2023:6134[OpenShift Logging Bug Fix Release 5.8.0 Kibana].
+
+[id="logging-release-notes-5-8-0-enhancements"]
+== Enhancements
+
+[id="logging-release-notes-5-8-0-log-collection"]
+=== Log Collection
+* With this update, you can deploy multiple, isolated, and RBAC-protected `ClusterLogForwarder` custom resource (CR) instances in any namespace. This allows independent groups to forward desired logs to any destination and also separates logging groups to interfere with another configuration. (link:https://issues.redhat.com/browse/LOG-1343[LOG-1343])
+
+* With this update, you can use the flow control or rate limiting mechanism to limit the volume of log data that can be collected or forwarded by dropping excess log records. The input limits prevent badly-behaved containers from overloading the {logging-uc} and the output limits put a ceiling on the amount of log data stored. (link:https://issues.redhat.com/browse/LOG-884[LOG-884])
+
+* With this update, a new supported small-scale LokiStack size of 1x.extra-small is introduced for OCP clusters hosting a few workloads and smaller ingestion volumes (upto 100GB/day). (link:https://issues.redhat.com/browse/LOG-4562[LOG-4562])
+
+* With this update, you can forward Kubernetes and OpenShift API server events in accordance with audit logging policies. The policies are now associated with audit log inputs in the `ClusterLogForwarder`. (link:https://issues.redhat.com/browse/LOG-3982[LOG-3982])
```

Comment:
With this update, you can configure audit polices to control which Kubernetes and OpenShift API server events are forwarded by the log collector.